software/rollershutter/ArduinoIDE/SmartSwitch.ino

```diff
@@ -397,11 +397,11 @@ void motionStop() {
   ausgabe = "STOP";                                                                   // Build strings to send to MQTT-Broker and send it
   top = topic + hostname_char + "/status/moving/";                                    // Built topic to sent message to
   client.publish(top.c_str(), ausgabe.c_str());                                       // Publish MQTT-Message
-  EEPROM.begin(4095);                                                                 // Define EEPROM
-  EEPROM.put(30, pos);                                                                // Write "pos" to EEPROM
-  delay(200);                                                                         // Delay
-  EEPROM.commit();                                                                    // Only needed for ESP8266 to get data written
-  EEPROM.end();                                                                       // Free RAM copy of structure
+  //EEPROM.begin(4095);                                                                 // Define EEPROM
+  //EEPROM.put(30, pos);                                                                // Write "pos" to EEPROM
+  //delay(200);                                                                         // Delay
+  //EEPROM.commit();                                                                    // Only needed for ESP8266 to get data written
+  //EEPROM.end();                                                                       // Free RAM copy of structure

```

Comment:
From my point of view, this four lines are necessarry becaus in case of power fail, without storing the stopping position to EEPROM, SmartSwitch doesn't know the position further more